addons/rmsmartshape/shapes/shape_base.gd

```diff
@@ -173,56 +172,53 @@ func get_point_array() -> SS2D_Point_Array:
 	return __points
 
 
-func set_point_array(a: SS2D_Point_Array, make_unique: bool = true):
+func set_point_array(a: SS2D_Point_Array, make_unique: bool = true) -> void:
 	if __points != null:
-		if __points.is_connected(
-			"material_override_changed", Callable(self, "_handle_material_override_change")
-		):
-			__points.disconnect(
-				"material_override_changed", Callable(self, "_handle_material_override_change")
-			)
+		if __points.is_connected("material_override_changed", self._handle_material_override_change):
+			__points.disconnect("material_override_changed", self._handle_material_override_change)
 	if make_unique:
+		# FIXME: This prevents clearing points resource in the editor and prints an error.
```

Comment:
Add an issue for this please

Reply:
There's actually several of such FIXMEs I left while doing the refactoring. I made notes about them in a text file. I could add those notes as an issue. Or should I separate them?

Reply:
I've only marked this one because it actually breaks functionality, while others are only unused members. You could add another issue to remove everything marked as unused.

Reply:
I'll make an issue then. I've tested on upstream 3.x, and it's also an issue there.

Reply:
Ah ok, so it's not our bug, so it doesn't have priority. That's good.